brownie/network/account.py

```diff
@@ -108,17 +108,25 @@ def load(self, filename: str = None) -> Union[List, "LocalAccount"]:
 
         Returns:
             Account instance."""
-        project_path = _get_data_folder().joinpath("accounts")
+        base_accounts_path = _get_data_folder().joinpath("accounts")
         if not filename:
-            return [i.stem for i in project_path.glob("*.json")]
+            return [i.stem for i in base_accounts_path.glob("*.json")]
+
         filename = str(filename)
-        if not filename.endswith(".json"):
-            filename += ".json"
         json_file = Path(filename).expanduser()
+
         if not json_file.exists():
-            json_file = project_path.joinpath(filename)
-            if not json_file.exists():
-                raise FileNotFoundError(f"Cannot find {json_file}")
+            temp_json_file = json_file.with_suffix(".json")
+            if temp_json_file.exists():
+                json_file = temp_json_file
+            else:
+                json_file_name = filename.split('/')[-1]
```

Comment:
```suggestion
                json_file_name = json_file.name
```

`Path` provides an attribute for this.  Manipulating the string with `/` could cause an issue on Windows filesystems.

Reply:
Ohhh, that is good, I will correct it now.